common/app/views/fragments/nav/newHeaderMenu.scala.html

```diff
@@ -146,4 +153,18 @@
             </ul>
         }
     </div>
+
+    <div class="menu-brand-extensions hide-until-desktop">
```

Comment:
Done in https://github.com/guardian/frontend/pull/17047/commits/694734ff766eed84e6e63ac2820b2b2610ebb371. Let's keep the attribute selector for now and see, whether we need some refactoring at the end if this becomes too much.